package/oxide/package

```diff
@@ -3,13 +3,13 @@
 # SPDX-License-Identifier: MIT
 
 pkgnames=(erode fret oxide rot tarnish)
-pkgver=2.0.1~beta-4
+pkgver=2.0.2~beta-1
 timestamp=2020-12-06T20:09Z
```

Comment:
timestamp ;)

Reply:
But I don't wanna :(

*sigh* I really hate this timestamp portion. Give me a sec.